index.d.ts

```diff
@@ -45,7 +45,7 @@ declare namespace tempy {
 declare const tempy: {
 	file: {
 		/**
-		The `callback` resolves with a temporary file path you can write to. The file is automatically cleaned up after the callback is executed.
+		The `callback` resolves with a temporary file path you can write to. The file is automatically cleaned up after the callback is executed. Returns a promise that resolves after the callback is executed and the file is cleaned up.
```

Comment:
Use the `@returns` doc keyword. Same in the other cases.

Reply:
Fixed!